src/components/splitlayout/examples/SplitLayoutExamples.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import { StepProgressBar } from '../../stepProgressBar/StepProgressBar';
+import { SplitLayout } from '../SplitLayout';
+import * as loremIpsum from 'lorem-ipsum';
+import { range } from 'underscore';
+
+const exampleTextLeft = `I am a left children. ${loremIpsum({ count: 5, sentenceUpperBound: 15 })}`;
```

Comment:
=> `children` is plural for `child` => `I am a left child` or `I am the left child`.

Same for the right one.

Reply:
my bad :D thanks for the typo